moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js

```diff
@@ -286,7 +287,10 @@ const ProjectSummary = ({ loading, error, data, refetch }) => {
               </ErrorBoundary>
             )}
           </Grid>
-          <Grid item xs={12}>
+          <Grid item xs={6}>
```

Comment:
This probably isn't a huge deal, but the Tags and Subprojects sections get pretty squished when they are next to each other on small screens. Maybe they could be set to 6 on medium or large and then 12 for small or xs to let them breathe some more?

Reply:
good call -- I updated it. Thank you!